docs-src/concepts/what-is-an-action.md

```diff
@@ -0,0 +1,46 @@
+---
+id: what-is-an-action
+title: What is an Action?
+sidebar_label: Action
+description: An Action is the fundamental pricing unit in Temporal Cloud. Each Action is an Event in the Event History of a Workflow Execution.
+tags:
+  - term
+  - explanation
+---
+
+An Action is the fundamental pricing unit in [Temporal Cloud](/concepts/what-is-temporal-cloud).
+Each Action is an [Event](/concepts/what-is-an-event) in the [Event History](/concepts/what-is-an-event-history) of a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+(Be aware that not all Events are Actions.)
+
+**Workflow Actions**
+
+- Start Workflow Execution.
+- Reset Workflow Execution.
+- Signal-With-Start Workflow Execution.
+- Continue-As-New Workflow Execution.
+- Start Child Workflow Execution.
+- Upsert Workflow Search Attributes.
+
+**Activity Actions**
+
+- Schedule Activity Task.
+- Activity Retry. Each retry is counted as a separate Action.
```

Comment:
I am a bit confused here. So just scheduling an activity that may never run is an action? But it's first attempt is not an action but each attempt after the first is? One might expect for simplicity each activity attempt is an action.